client/src/components/Icon/index.js

```diff
@@ -12,14 +13,15 @@ const Icon = function iconBuilder(props) {
   switch (iconClass) {
     case 'fa': return (<FontAwesome {...props} name={iconName} />);
     case 'mci': return (<MaterialCommunityIcons {...props} name={iconName} />);
+    case 'mi': return (<MaterialIcons {...props} name={iconName} />);
     case 'fnd': return (<Foundation {...props} name={iconName} />);
     case 'oi': return (<Octicons {...props} name={iconName} />);
 
     default: return (<FontAwesome {...props} name={props.name} />);
   }
 };
 Icon.propTypes = {
-  name: PropTypes.string,
+  props: PropTypes.string,
```

Comment:
no pretty sure this should be name. The other props should be listed too but eslint isn't smart enough to require you to add them. It's fine to leave them out imo.